postcss.config.js

```diff
@@ -0,0 +1,7 @@
+const tailwindcss = require('tailwindcss');
+module.exports = {
+    plugins: [
```

Comment:
Would you be able to format this file as well?

Reply:
I based the ESLint PR off this webpack branch so will need to merge this PR first and then I will make sure the formatting is correct in that PR :)